zppy/templates/mpas_analysis.bash

```diff
@@ -139,6 +139,42 @@ parallelTaskCount = {{ parallelTaskCount }}
 # handle 12 simultaneous processes, one for each monthly climatology.
 ncclimoParallelMode = bck
 
+# the number of total threads to use when ncclimo runs in "bck" or "mpi" mode.
+# Reduce this number if ncclimo is crashing (maybe because it is out of memory).
+# The number of threads must be a factor of 12 (1, 2, 3, 4, 6 or 12).
+ncclimoThreads = 12
+
+# the number of MPI tasks to use in creating mapping files (1 means tasks run in
+# serial, the default)
+{% if machine == 'cori' and partition == 'knl' %}
+# Cori-KNL does not support creating mapping files in parallel
+mapMpiTasks = 1
+{% elif machine in ['cori', 'compy', 'anvil', 'chrysalis'] %}
+mapMpiTasks = {{ mapMpiTasks }}
+{% endif %}
+
+# "None" if ESMF should perform remapping in serial without a command, or one of# "srun" or "mpirun" if it should be run in parallel  (or in serial but with a
+# command)
```

Comment:
Looks like you meant to have this be three lines of comments?

Reply:
```suggestion
# "None" if ESMF should perform remapping in serial without a command, or one of
# "srun" or "mpirun" if it should be run in parallel  (or in serial but with a
# command)
```

Reply:
@forsyth2, Good catch!  I messed this one up in the terminal window and copied it a bunch of places, including here, before I noticed.